dev.Dockerfile

```
@@ -1,18 +1,19 @@
-FROM --platform="linux/arm64" arm64v8/node@sha256:56e8282f4392fb96c877babc93b3829e46b79c6fbcd48c92de578febffc80587 AS builder
+ARG NODE_SHA=sha256:56e8282f4392fb96c877babc93b3829e46b79c6fbcd48c92de578febffc80587
+ARG TARGET_PLATFORM=linux/arm64
```

```suggestion
ARG TARGET_PLATFORM=linux/arm64
```
The platform switch can be removed since this image is locked to `arm64` due to the image prefix `arm64v8/node`